### PR TITLE
[stable/wordpress] Fix ingress metadata.name bug.

### LINKED ISSUE #2920: [stable/wordpress] Ingress bad metadata name with Google Cloud
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
helm: v2.7.0
kube: v1.8.4-gke.0

**Which chart**:
stable/wordpress (0.7.5)

**What happened**:
`googleapi: Error 400: Invalid value for field 'resource.name': 'k8s-um-blog-wp001-blog.example.com-wp001--d8aaf002fa3d533e'. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)', invalid`

**How to reproduce it** (as minimally and precisely as possible):
Enable ingress, and use Google Cloud.

**Anything else we need to know**:
It could be good to manage a dedicated value for the ingress name if enabled.

### PR DESCRIPTION
**Pull request by ismailbaskin**
It simply fixes https://github.com/kubernetes/charts/issues/2920 issue.

### CODE REVIEW

**Comment by ismailbaskin** [comment]
CircleCI tests fail because of the wrong allowEmptyPassword value in values.yaml . Should I fix it in this PR or another?

**Comment by sameersbn** [comment]
> CircleCI tests fail because of the wrong allowEmptyPassword value in values.yaml . 

I don't think this is the case here. Could you revert this change and recheck.

File: stable/wordpress/templates/ingress.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | replace "." "_" | trimSuffix "-" -}}"
```
Comment (prydonius) [inline stable/wordpress/templates/ingress.yaml:6]: `_` isn't valid right? Should probably be `-`.

File: stable/wordpress/values.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ wordpressBlogName: User's Blog!
 
 ## Set to `yes` to allow the container to be started with blank passwords
 ## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
-allowEmptyPassword: yes
+allowEmptyPassword: true
```
Comment (prydonius) [inline stable/wordpress/values.yaml:44]: we should leave this as-is

File: stable/wordpress/templates/ingress.yaml (hunk 1 above)
Comment (ismailbaskin) [reply stable/wordpress/templates/ingress.yaml:6]: I think domain dots must be different from other parts.

File: stable/wordpress/values.yaml (hunk 2 above)
Comment (ismailbaskin) [reply stable/wordpress/values.yaml:44]: is there any reason?

File: stable/wordpress/templates/ingress.yaml (hunk 1 above)
Comment (prydonius) [reply stable/wordpress/templates/ingress.yaml:6]: What I mean is that this replaces names like `test.test` to `test_test` which is also an invalid name in Kubernetes. `test-test` is valid, though, so we should replace the dots with `-`.

File: stable/wordpress/values.yaml (hunk 2 above)
Comment (juan131) [reply stable/wordpress/values.yaml:44]: This environment variable is used by the WP [Nami](https://github.com/bitnami/nami) module which won't recognised "true" as a valid value.

Valid values are: ["yes", "no"]

**Review by juan131 — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
What's the status with this?

**Comment by juan131** [comment]
HI @unguiculus  we're waiting for @ismailbaskin to update the PR so he uses the right value for **allowEmptyPassword**. _true_ is not a valid value.

**Comment by ismailbaskin** [comment]
Closed in favor of https://github.com/kubernetes/charts/pull/5550

**Pull request closed without merging**